docs/components/integration-framework/connectors/out-of-the-box-connectors/rest.md

```diff
@@ -6,6 +6,9 @@ description: Make a request to a REST API and use the response in the next steps
 
 The **REST Connector** allows you to make a request to a REST API and use the response in the next steps of your process.
 
+If you see that connector mark as 'Deprecated', you can update and replace it.
+See [Replacing Templates](../../../../components/modeler/desktop-modeler/element-templates/using-templates.md) for more.
+
```

Comment:
@Oleksiivanov no need to add these sentence to REST Connector docs. 
You can remove entire: 

"If you see that connector mark as 'Deprecated', you can update and replace it.
See [Replacing Templates](../../../../components/modeler/desktop-modeler/element-templates/using-templates.md) for more."